cmd/hlb/command/run.go

```diff
@@ -7,6 +7,7 @@ import (
 	"os"
 	"strings"
 
+	"github.com/containerd/console"
```

Comment:
It seems like we are importing this package simply for an interface.  I vote for just adding out own interface here:
```
type File interface {
	io.ReadWriteCloser
	Fd() uintptr
	Name() string
}
```

Reply:
https://github.com/openllb/hlb/pull/232